### ISSUE TITLE
Translator load error

### ISSUE THREAD

**Issue by phlonkyou** [Amulet-Team/Amulet-Map-Editor#313]
**Has this bug already been reported? Please check if there is a bug report already open that matches this.**

**Describe the bug**

when i go to load a world in the 3d editor, it just gets stuck loading the resource pack halfway through. no other issues loading the world beforehand or any options besides that.

**To Reproduce**

so what happened to me was that i loaded into a new 1.17 world using 0.8.4 amulet. i initially got a popup saying "Failed to download the latest Java resource pack. Check your internet connection and restart Amulet. Check the console for more details." it continued to load into the world, however it was pretty much just a void because i was using an outdated amulet version for the new world so maybe the new blocks would have caused issues. i realized that pretty much immediately and then i updated to the newest version. now when i load into any world in the newest version, it just stops loading the resource pack halfway through the loading screen and nothing happens. 

**Expected behavior**

i figured the problem would be resolved if i reinstalled the latest version but it didnt and i am stuck with this issue

**Screenshots**

https://imgur.com/a/bt43zcz
https://i.imgur.com/LXfBFuD.png
^ this screenshot isnt mine, this is someone else's in the discord who had a similar issue. i used this because its the same popup i got but i didnt screenshot. the error in the last line is specific to their issue. 

**Desktop (please complete the following information):**
windows 10 version 20h2 (os build 19042.985)

**Additional context**

**Comment by Kiernan531** [comment]
I have the same problem.  Had the error message when using the last version.  Updated to new version and when opening 3d editor it just freezes halfway through.  The console says there is an error finding a specified path but when I look for the file listed it does exist in that folder.

![image](https://user-images.githubusercontent.com/85450885/121264177-bf8f2600-c884-11eb-94ca-ce53c8ba309e.png)

**Comment by phlonkyou** [comment]
i may have inadvertently found a fix for this problem. so i have been using amulet for a few patches, and each time i update it, i extract the contents of the new zip into the same Amulet folder. this has caused no issues for the past 3-4 patches, but when i created a new folder to extract the contents of 0.8.7 into, it booted up and loaded 3d editor with no problems. maybe this issue is specific to the files of 0.8.4 and 0.8.7, but regardless, this worked for me.

**Comment by gentlegiantJGC** [comment]
It sounds like a file has been extracted to somewhere it should not have been.

**Comment by gentlegiantJGC** [comment]
If someone that has this issue can zip up their Amulet directory and send it to me I will look into what is causing it.

**Comment by Podshot** [comment]
It looks like the Amulet folder is located in the directory tree of the Bedrock installation, try moving the `Amulet` directory to your desktop and running it from there

**Comment by gentlegiantJGC** [comment]
There was another user in the discord server that had the same error.
Extracting to a new directory solved the problem but I don't know why.
I haven't changed that part of the code in probably the last year which is why I want an example to debug what is causing it

**Comment by gentlegiantJGC** [comment]
The issue @Kiernan531 has reported is also a different issue to the original report

**Comment by Podshot** [comment]
Oh I see, I didn't notice that it was `TypeError` in the first screenshot and not a `FileNotFoundError` as in the second one

**Comment by gentlegiantJGC** [comment]
The first one is an issue in the translator and the second on is in the model library. It is coming from unzipping the assets so I don't know why it would be doing that.

**Comment by phlonkyou** [comment]
the second screenshot is from an error someone else had in the discord. i got the same popup with a different error text at the bottom of it. i used this screenshot because i didnt screenshot the error when i got it

**Comment by phlonkyou** [comment]
also the error they had was described the same as what i had but they fixed it by reinstalling also

**Comment by gentlegiantJGC** [comment]
Yeh I am aware of the pack.mcmeta issue which is an issue with the old version

**Comment by gentlegiantJGC** [comment]
A user in the discord server sent me their files.
The issue is from version 139. Not entirely sure what caused it.
It looks like an issue with the build process though

**Comment by gentlegiantJGC** [comment]
Okay I know what the issue was.
I removed one of the bedrock versions from the translator because they did not update the block version in 1.17 so I removed the old one to remove the conflict.
The issue is that you did not delete the old one so it had a mixture of the old and new data.
I should add a way to detect this but currently the only fix is to delete the Amulet directory and extract again.
